meta/sai_serialize.h

```diff
@@ -278,4 +279,8 @@ void sai_deserialize_queue_attr(
         _In_ const std::string& s,
         _Out_ sai_queue_attr_t& attr);
 
+void sai_deserialize_media_change_ntf(
```

Comment:
I am not sure if we have such ntf? can you point out the sai function for this?

Reply:
I think the name of the API might be little misleading here compared to other APIs present. The ntf stands for media notification. The intent of the API is to deserialize the notification received from xcvrd into list of SAI attributes. There are three attributes handled currently as mentioned in the above comment and in future many can be added. Thus i intended to keep the name of API as deserialize media notificaiton.
Please let me know if you want to rename the API